static/app/utils/replays/replayReader.tsx

```diff
@@ -144,16 +145,18 @@ function removeDuplicateNavCrumbs(
 
 const extractDomNodes = {
   shouldVisitFrame: frame => {
-    const nodeId = getNodeId(frame);
-    return nodeId !== undefined && nodeId !== -1;
+    const nodeIds = getNodeIds(frame);
+    return nodeIds.filter(nodeId => nodeId !== -1).length > 0;
   },
   onVisitFrame: (frame, collection, replayer) => {
     const mirror = replayer.getMirror();
-    const nodeId = getNodeId(frame);
-    const html = extractHtml(nodeId as number, mirror);
+    const nodeIds = getNodeIds(frame);
+    const html = extractHtml(nodeIds as number[], mirror);
```

Comment:
`as number[]` could be more dangerous than the `as number` that we had before. If we pass `undefined` into here it looks like we'll try to iterate over it which could explode.

Maybe this needs to be something like `(nodeIds ?? []) as number[]`